code/PostgreSQLSchemaManager.php

```diff
@@ -537,8 +543,8 @@ private function alterTableAlterColumn($tableName, $colName, $colSpec)
                 if ($this->hasTable("{$tableName}_Live")) {
                     $updateConstraint .= "UPDATE \"{$tableName}_Live\" SET \"$colName\"='$default' WHERE \"$colName\" NOT IN ($constraint_values);";
                 }
-                if ($this->hasTable("{$tableName}_versions")) {
-                    $updateConstraint .= "UPDATE \"{$tableName}_versions\" SET \"$colName\"='$default' WHERE \"$colName\" NOT IN ($constraint_values);";
+                if ($this->hasTable("{$tableName}_Versions")) {
```

Comment:
Don't we have constants for these suffixes?

Reply:
Nah ... their hard coded in Versioned.